pylint/test/functional/consider_merging_isinstance.txt

```diff
@@ -0,0 +1,5 @@
+consider-merging-isinstance:15:isinstances:Consider merging these isinstance calls to var[3].
```

Comment:
Why would it require a full refactoring? It could be done in this PR, we will merge it after it.